### PR TITLE
simplify issue template

### PR DESCRIPTION
**Pull request by Lijiaoa**
https://github.com/microsoft/nni/issues/2518

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -29,11 +25,13 @@ Please fill this for deployment related issues:
 - is conda or virtualenv used?: 
 - is running in docker?:
 
+**Log message**:
+ - nnimanager.log: 
+ - dispatcher.log:
+ - nnictl stdout and stderr:
+ 
+reference for these file: [log](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director), [stdout/stderr](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout)
```
Comment (QuanluZhang) [inline .github/ISSUE_TEMPLATE/bug-report.md:33]: Where can you find the log files: https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -5,35 +5,27 @@ about: Report an issue or question while using nni instance (deployment).
...
+- PyTorch/TensorFlow version:
+- Is conda/virtualenv/venv used?:
+- Is running in Docker?:
 
+**Log message**:
+ - nnimanager.log: 
+ - dispatcher.log:
+ - nnictl stdout and stderr:
+ 
+Where can you find the log files: [log](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director), [stdout/stderr](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout)
```
Comment (liuzhe-lz) [inline .github/ISSUE_TEMPLATE/bug-report.md:23]: Do NOT use hyperlink (`[x](xxx)`) in template file.

Comment (Lijiaoa) [reply .github/ISSUE_TEMPLATE/bug-report.md:23]: OK. will change it to

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -5,35 +5,27 @@ about: Report an issue or question while using nni instance (deployment).
...
+- Is running in Docker?:
 
+**Log message**:
+ - nnimanager.log: 
+ - dispatcher.log:
+ - nnictl stdout and stderr:
+ 
+<!-- Where can you find the log files: [log](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director), [stdout/stderr](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout) -->
 
 **Short summary about the issue/question**:
```
Comment (squirrelsc) [inline .github/ISSUE_TEMPLATE/bug-report.md:25]: **What issue meet, what's expected?**

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -5,35 +5,27 @@ about: Report an issue or question while using nni instance (deployment).
...
 
 **Brief what process you are following**:
```
Comment (squirrelsc) [inline .github/ISSUE_TEMPLATE/bug-report.md:27]: **How to reproduce it?**

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 5]
```diff
@@ -5,35 +5,27 @@ about: Report an issue or question while using nni instance (deployment).
...
-
-**nni Environment**:
-- nni version:
-- nni mode(local|pai|remote):
-- OS:
-- python version:
-- is conda or virtualenv used?: 
-- is running in docker?:
-
 **need to update document(yes/no)**:
```
Comment (squirrelsc) [inline .github/ISSUE_TEMPLATE/bug-report.md:29]: It looks not worth to be a question, and can be removed.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 6]
```diff
@@ -5,35 +5,27 @@ about: Report an issue or question while using nni instance (deployment).
...
 
 **Anything else we need to know**:
```
Comment (squirrelsc) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]: **Additional information**

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 7]
```diff
@@ -5,35 +5,27 @@ about: Report an issue or question while using nni instance (deployment).
 
 ---
 
-<!-- Please use this template while reporting an issue and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->
+**NNI Environment**:
```
Comment (squirrelsc) [inline .github/ISSUE_TEMPLATE/bug-report.md:8]: Environment

**Pull request merged**